### DISCUSSION TITLE
Cursor not showing up on osu!lazer 2023.511.0

### DISCUSSION THREAD

**Discussion by L1ght7ear2020YT** [ppy/osu#23503] [Q&A]
### Type

Other

### Bug description

My cursor is not showing up. WHY???

### Screenshots or videos

![osu_2023-05-11_12-46-44](https://github.com/ppy/osu/assets/95110117/83ba86b3-2842-4c07-9480-0ad0fbad3610)

### Version

2023.511.0

### Logs

N/A

**Comment by bdach** [comment]
I can't reproduce this, and I don't know how to debug this without more info, especially that logs are missing. At the least:

- What OS are you running?
- What are the specs of your device?
- What renderer are you currently using?

**Comment by Sinamer** [comment]
The disappearing is seemingly random, but I found out that taking a screenshot makes it quite consistent to happen (probably due to lag spike caused by taking it)
Happens on OpenGL (Legacy), other renderers I can't check due to them crashing currently

Here's a video of it happening:

https://github.com/ppy/osu/assets/53346335/8241f661-9e7b-4137-8b1f-132b310fd014



I took RenderDoc captures before and after the cursor disappearing just in case:
https://drive.google.com/file/d/15WJT0xhReciOpWz8uz21PeCYVhJvbLcW/view?usp=share_link

Version:
Framework and osu! master branches

Logs:
[runtime.log](https://github.com/ppy/osu/files/11468890/runtime.log)
[input.log](https://github.com/ppy/osu/files/11468891/input.log)
[network.log](https://github.com/ppy/osu/files/11468892/network.log)
[performance.log](https://github.com/ppy/osu/files/11468889/performance.log)

**Reply by bdach** [reply]
Thanks, that's helpful.

I'm not sure that this is the same issue that the OP has, because it is an issue with the screenshot-taking flow specifically, but it does appear to be an actual reproducible issue.

**Reply by bdach** [reply]
Tracking in https://github.com/ppy/osu/issues/23523

**Comment by bdach** [comment]
@L1ght7ear2020YT has the latest hotfix release fixed this for you?
